packages/frontend/src/pages/routes/UnauthenticatedRoutes.tsx

```diff
@@ -10,6 +11,9 @@ const UnauthenticatedRoutes: React.FC<UnauthenticatedRoutesProps> = () => {
   return (
     <Routes>
       <Route path="auth" element={<SignInPage />} />
+      <Route path="home" element={<HomePage />} />
+      <Route path="mgmt" element={<AccountMgmtPage />} />
```

Comment:
can this be `/account` and `<ManageAccountPage />`

Reply:
Ok